pynamodb/connection/base.py

```diff
@@ -6,6 +6,7 @@
 import logging
 import math
 import random
+import requests
```

Comment:
you'll want to add `requests` to `setup.py`'s `install_requires`

Reply:
Ya, that's a blunder. Fixed it.